website/docs/flink-quick-start-guide.md

```diff
@@ -322,6 +359,20 @@ The `DELETE` statement is supported since Flink 1.17, so only Hudi Flink bundle
 Only **batch** queries on Hudi table with primary key work correctly.
 :::
 
+</TabItem>
+
+<TabItem value="dataStream">
+
+Creates a Flink Hudi table first and insert data into the Hudi table using DataStream API as below.
+When new rows with the same primary key and Row Kind as Delete arrive in stream, then it will be be deleted.
+
+Refer Delete Example [here](https://github.com/ad1happy2go/hudi-examples/blob/main/flink/src/main/java/com/hudi/flink/quickstart/HudiDataStreamReader.java)
```

Comment:
Are the examples complete yet ?

Reply:
It's complete but it was pointing to wrong code. Fixed it.